prac_05/hex_colours.py

```diff
@@ -3,6 +3,7 @@
 Estimate: 20 minutes
 Actual:   25 minutes
 """
+
 NAME_TO_CODE = {"Absolute Zero": "#0048ba", "Acid Green": "#b0bf1a", "AliceBlue": "#f0f8ff", "Alizarin crimson": "#e32636", "Amaranth": "#e52b50", "Amber": "#ffbf00", "Amethyst": "#9966cc", "AntiqueWhite": "#faebd7", "Ao": "#0000ff", "Apple Green": "#8db600"}
```

Comment:
"AliceBlue" & "AntiqueWhite" won't match with a user input that is operated with title() method. You have to use "Alice Blue" etc.